scripts/apps/monitoring/controllers/MonitoringController.js

```diff
@@ -13,15 +13,25 @@ export function MonitoringController($rootScope, $location, desks) {
     this.viewSingleGroup = viewSingleGroup;
     this.viewMonitoringHome = viewMonitoringHome;
 
+    this.getSwimlaneView = config.features && config.features.swimlane ? 1 : 0;
```

Comment:
I would change this name, `getSwimlaneView` sounds like a function that is "_getting_" something. While this is more like a check, so something like `hasSwimlaneView` would be more appropriate.